WiinUPro/Windows/DevicePrefsWindow.xaml.cs

```diff
@@ -24,14 +25,14 @@ public partial class DevicePrefsWindow : Window
         private DevicePrefs _modifiedPrefs;
 
         private int lastIndex;
-        private bool selectionChanged = false;
+        private bool extFunc = false;
 
         protected DevicePrefsWindow()
         {
             InitializeComponent();
         }
 
-        public DevicePrefsWindow(DevicePrefs devicePrefs, string type = null) : this()
+        public DevicePrefsWindow(DevicePrefs devicePrefs, ControllerType type = ControllerType.Other) : this()

```

Comment:
This might pose a problem if an unknown Wiimote extension is connected, since the extension specific profile menu would be hidden.

Reply:
True, though not a big deal since they could just unplug it.
You can set extFunc to True if the type is not Other or ProController.